src/NoteField.cpp

```diff
@@ -867,7 +867,15 @@ void NoteField::DrawPrimitives()
 		ASSERT(GAMESTATE->m_pCurSong != nullptr);
 
 		const TimingData &timing = *pTiming;
-		const RageColor text_glow= RageColor(1,1,1,std::cos(RageTimer::GetTimeSinceStartFast()*2)/2+0.5f);
+
+		// Create an oscillating / pulsing glow effect.
+		// Converts a counter to radians and uses the cosine for a cyclic appearance.
+		// Ideally, m_iGlowCounter shouldn't exceed 360 (hence uint8_t).
```

Comment:
To guarantee this, why do we not use modulo?

Reply:
> To guarantee this, why do we not use modulo?

The modulo is not needed because it's impossible for uint8_t to exceed 255, which is perfect here because as long as the number stays below 360 the function works as intended.  It still produces the same cyclic effect that we need to accomplish.

Reply:
Wouldn't we want it to be 360 exactly otherwise it would *skip* from 255 to 0 instead of doing a full rotation?

Reply:
I personally can't see any difference between the original code vs using a 360 modulo vs letting a uint8_t wrap at 255.  So i decided to go with uint8_t because it only consumes a single byte. 

With that said, it's not like it's the end of the world if we use a few more bytes so I will reinstate the modulo.